certification/pyxis/submit.go

```diff
@@ -0,0 +1,54 @@
+package pyxis
+
+import (
+	"context"
+	"strings"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+func (p *pyxisEngine) SubmitResults(containerImage string) error {
+	ctx := context.Background()
+	projectId := p.ProjectId
+	if projectId == "" {
+		return errors.ErrEmptyProjectID
+	}
+	if strings.HasPrefix(projectId, "ospid-") {
+		projectId = strings.Split(projectId, "-")[1]
+	}
+	project, err := p.getProject(ctx)
```

Comment:
there's a task for displaying the response from Pyxis to the user - https://github.com/redhat-openshift-ecosystem/openshift-preflight/issues/376
can we add it here?

Reply:
We don't have that value in the `struct` yet so we can implement #376 separately. I think we might want to have SubmitResults, return that value, then the man `cmd` can display it before it exits.